src/main/java/org/opensrp/repository/ManifestRepository.java

```diff
@@ -8,12 +8,21 @@
 
 public interface ManifestRepository extends BaseRepository<Manifest> {
 
-	@Nullable
-	Manifest getManifestByAppId(String appId);
+    @Nullable
+    Manifest getManifestByAppId(String appId);
```

Comment:
This formatting should use one here https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/253034611/How+to+add+edit+remove+a+table+column. I tried reformatting on my local and it tells me no line changed

Reply:
I also tried with the same formatter and it told me the same too. Not sure why.. though

Reply:
https://smartregister.atlassian.net/wiki/spaces/Documentation/pages/1478721546/Importing+code+formatter+in+Intellij+Community+Edition